### PR TITLE
Fix or Disable lint wherever needed

### PR DESCRIPTION
**Pull request by PushkarJ**

### CODE REVIEW

**Comment by PushkarJ** [comment]
> [✖] https://eba.europa.eu/documents/10180/2170121/Final+draft+Recommendations+on+Cloud+Outsourcing+%28EBA-
> Rec-2017-03%29.pdf → Status: 404

This is a false positive CI failure as the link opens if you click on it :)

File: security-whitepaper/v2/cloud-native-security-whitepaper.md [hunk 1]
```diff
@@ -816,7 +816,7 @@ Registries should accommodate technologies to sign and verify OCI artifacts. It
 caching and distribution tools also provide the capability to sign, encrypt and provide checksums to ensure that the
 caching layer can detect tampering or attempts to poison the dataset.
 
-The[ CNCF Storage Whitepaper](https://bit.ly/cncf-storage-whitepaperV2) provides additional background on the concepts,
+The[CNCF Storage Whitepaper](https://bit.ly/cncf-storage-whitepaperV2) provides additional background on the concepts,
```
Comment (TheFoxAtWork) [inline security-whitepaper/v2/cloud-native-security-whitepaper.md:819]:
```suggestion
The [CNCF Storage Whitepaper](https://bit.ly/cncf-storage-whitepaperV2) provides additional background on the concepts,
```

File: security-whitepaper/v2/cloud-native-security-whitepaper.md [hunk 2]
```diff
@@ -919,13 +919,13 @@ For organizations adopting cloud native, a primary mechanism for identifying ris
 perform threat modeling. While there are many threat modeling techniques, they share several core characteristics. All
 start with building a scoped representation of a system’s architecture. This begins with identifying all important
 processes, data stores,
-and[ security boundaries](https://www.oreilly.com/library/view/cissp-certified-information/9780470276884/9780470276884_security_boundaries.html)
+and[security boundaries](https://www.oreilly.com/library/view/cissp-certified-information/9780470276884/9780470276884_security_boundaries.html)
```
Comment (TheFoxAtWork) [inline security-whitepaper/v2/cloud-native-security-whitepaper.md:922]:
```suggestion
and [security boundaries](https://www.oreilly.com/library/view/cissp-certified-information/9780470276884/9780470276884_security_boundaries.html)
```

File: security-whitepaper/v2/cloud-native-security-whitepaper.md [hunk 3]
```diff
@@ -919,13 +919,13 @@ For organizations adopting cloud native, a primary mechanism for identifying ris
...
 . Once boundaries have been established and the relevant elements of the system are partitioned within them, the next
 step is to model how these elements interact, with special attention paid to any interactions that cross security
 boundaries.
 
 The below guidance is an enhancement of the
-four-step[ OWASP threat modeling](https://owasp.org/www-community/Threat_Modeling) recommended for cloud native
+four-step[OWASP threat modeling](https://owasp.org/www-community/Threat_Modeling) recommended for cloud native
```
Comment (TheFoxAtWork) [inline security-whitepaper/v2/cloud-native-security-whitepaper.md:928]:
```suggestion
four-step [OWASP threat modeling](https://owasp.org/www-community/Threat_Modeling) recommended for cloud native
```

**Review by TheFoxAtWork — CHANGES_REQUESTED** [review]
Looks like the space in the link brackets should have been before the bracket.

**Review by TheFoxAtWork — APPROVED** [review]
LGTM thanks @PushkarJ

**Comment by TheFoxAtWork** [comment]
@PushkarJ seems like there is one broken link for EBA.  I leave it to you if you want to merge this in, otherwise it looks great to me.

**Comment by PushkarJ** [comment]
@TheFoxAtWork thank you! I replaced with a link that loaded fast enough for link checker to turn green!

**Pull request merged**